### ISSUE TITLE
[ui_next] Allow the user to navigate between items on a modal list with keyboard navigation

### ISSUE THREAD

**Issue by unlikelyzero** [ansible/awx#8859]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
I'm not sure if this is an accessibility violation, but feedback from the bug bash:

```I love being able to tab to different elements in the UI!  I have no idea how possible this is, but it would be cool to be able to use the arrow key to move between items in the filtered lists, for example, for Projects on a JT.  ```

##### ADDITIONAL INFORMATION

<img width="637" alt="Screen Shot 2020-12-10 at 6 44 50 PM" src="https://user-images.githubusercontent.com/12446869/101843260-d0e6b800-3b17-11eb-9f7f-2323e2f028a5.png">

**Comment by wenottingham** [comment]
... this works fine now. Closing!
